### PR TITLE
Bjorncs/use bouncycastle node admin

### PR DESCRIPTION
**Pull request by bjorncs**
FYI @hakonhall @tokle

### CODE REVIEW

**Comment by bjorncs** [comment]
FYI @gjoranv

**Review by freva — APPROVED** [review]

File: docker-api/pom.xml [hunk 1]
```diff
@@ -122,6 +122,29 @@
                 <artifactId>bundle-plugin</artifactId>
                 <extensions>true</extensions>
             </plugin>
+            <plugin>
```
Comment (hakonhall) [inline docker-api/pom.xml]: why is this needed?

Comment (bjorncs) [reply docker-api/pom.xml]: To allow building `zone-application` without downloading snapshot releases from artifactory :)

Comment (hakonhall) [reply docker-api/pom.xml]: So mbuild builds something in addition to other than mvn, or renames files?
